dds_web/templates/mail/project_release.html

```diff
@@ -18,22 +18,28 @@
         </ul>
       </p>
       <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">
+        You were added to this project on behalf of {{displayed_sender}}.
+      </p>
+      <p style="font-family: arial, sans-serif; font-size: 12px; font-weight: normal; margin: 0; Margin-top: 0px;">
+        To list the files in this project, run: <br>
+        <i><mark id="ProjectListing">dds ls -p {{project_id}}</mark></i>. <button onclick="copyTextProjectRelease(1)">Copy command</button></p>
```

Comment:
Use the code block here instead. Which means you'll need to change the css formatting `mark` part in the mail_base.html file too.
```suggestion
        <code id="ProjectListing">dds ls -p {{project_id}}</code> <button onclick="copyTextProjectRelease(1)">Copy command</button></p>
```